### ISSUE TITLE
Genesis: import engagement group data tool

### ISSUE THREAD

**Issue by alpe** [confio/tgrade#72]
Maybe a CLI command that reads from a CSV file. This needs to be refined /cc @ks-victor

We need to track:

* engagement point holders (address,points)
* TGD balances (address,tokens) - Also which are vesting and what schedule
* list of OC members
* list of AP members

These could be eg 4 different files. They could be combined in one. Whatever makes best sense for you.

**Comment by ethanfrey** [comment]
How do we do this now? We have some way to set these, right? A JSON array in genesis at least?

I assume this is a process to take a csv file (exported from some Excel doc from Martin) and add it to the genesis file?

This may be very important for setting up the larger testnet and mainnet, can you please clarify the desired changes?

**Comment by ks-victor** [comment]
I guess that all validators still need at least 1 EP in order to validate, right?

What was the end result here of the EPs? We talked about a max of 1m and do we have a minimum that we should stay within? If so, I would for starters just take that minimum and spread it equally between the genesis validators.

**Comment by kmw101** [comment]
I will have a spreadsheet which can be saved as CSV.

How about we start all validators on 5,000 EP which equates to 350,000

**Comment by ethanfrey** [comment]
@ks-victor @kmw101 please move this discussion to https://github.com/confio/tgrade-testnets/issues/48 which discusses how we will set up the actual genesis file for the testnet.

This is discussing tooling. If we want a tool that takes a csv of addresses and engagement points and inserts it into the genesis file, that is a clear decision here. Do we need something similar for importing account balances? Although I guess there will be more manual setting for the vesting accounts?

Basically, what format will you be providing what data - then we can built tools to easily set up the genesis file.
Who gets what where is best done in the testnet issue

**Comment by BirgitPohl** [comment]
@daniellarita Please, check on what the status is. 
@kmw101 wants to provide a CSV file
and this discussion shall be moved to `tgrade-testnet`
https://github.com/confio/tgrade-testnets/issues/45
https://github.com/confio/tgrade-testnets/issues/46
https://github.com/confio/tgrade-testnets/issues/47
https://github.com/confio/tgrade-testnets/issues/48
https://github.com/confio/tgrade-testnets/issues/49
are mentioning the genesis file.

My question is, should this one be closed then?
@alpe
